aten/src/ATen/native/cuda/AveragePool2d.cu

```diff
@@ -21,14 +21,15 @@ __device__ inline int min(int a, int b) {
 __device__ inline int max(int a, int b) {
   return a >= b ? a : b;
 }
-
-template <typename scalar_t, typename accscalar_t, bool COUNT_INCLUDE_PAD, bool USE_DIVISOR>
+  
```

Comment:
Empty white space.
do a clang-format on this file. I do see CI complain on this.

Reply:
fixed